### PR TITLE
Adding hidden_value in smart class parameters and variables

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _includes/manuals/1.11/4.2.5_param_classes.md [hunk 1]
```diff
@@ -39,10 +39,12 @@ Lets configure the *foreman* class to change the user the foreman processes run
    * The type of data we want to pass. Most commonly a string, but many other data types are supported. There's no easy way to tell what type of data Puppet is expecting, so you will need to read through the code/documentation that comes with a particular module to find out. Changing the type field requires an appropriately set "Default Value" field.
 * Default Value
    * This will be imported from Puppet initially, but if Puppet is using any class inheritance, you'll get something unhelpful like "${$foreman::params::user}". This is because Foreman won't follow the inheritance, so you'll need to set a sensible default value
+* Hidden value
+   * Should the values of the smart class parameters be hidden.
```
Comment (tbrisker) [inline _includes/manuals/1.11/4.2.5_param_classes.md]: "..be hidden in the UI."
Also, s/parameters/parameter

File: _includes/manuals/1.11/4.2.4_smart_vars.md [hunk 2]
```diff
@@ -31,6 +31,10 @@ Click New Variable, and you will be presented with a set of input fields:
       <td>Default Value</td>
       <td>Value supplied to the ENC if no other criteria is matched</td>
     </tr>
+    <tr>
+      <td>Hidden value</td>
+      <td>Should the values of the smart variable be hidden</td>
```
Comment (tbrisker) [inline _includes/manuals/1.11/4.2.4_smart_vars.md]: ..in the UI

File: _includes/manuals/1.11/4.2.5_param_classes.md [hunk 3]
```diff
@@ -71,11 +73,15 @@ see the [Smart Matchers](manuals/{{page.version}}/index.html#4.2.6SmartMatchers)
 
 If Foreman manages the value of a class parameter ("override = true"), it's also possible to update a host-specific override from the host itself. That way you don't have to grant access to the Puppet Classes page to everyone. From a Host, click Edit, go to the Parameters tab, and you'll see the variable, the class-scope, and the current value. You can then override the value for that host:
 
-![Host Edit](/static/images/screenshots/param_classes_110/hostedit.png)
+![Host Edit](/static/images/screenshots/param_classes/1.11/hostedit.png)
+
+If the value is hidden you can click the unhide button to temporarily see the value while you edit. Clicking the button won't change the hidden property for the parameter, only show it for edit purpose.
```
Comment (tbrisker) [inline _includes/manuals/1.11/4.2.5_param_classes.md]: editing

**Comment by orrabin** [comment]
@tbrisker thanks, fixed your comments.

**Comment by domcleal** [comment]
Thanks @orrabin!

**Pull request merged**